netkan/setup.py

```diff
@@ -38,9 +38,6 @@
             'pylint',
             'autopep8',
             'troposphere',
-        ],
-        'test': [
-            'mypy',
         ]
```

Comment:
Our package should be explicit about its test dependencies.

Reply:
Mypy isn't needed to run unittest. I think we should handle such things individually, so we only have the tools we need in each case.

Reply:
`mypy` is technically a test dependency, but you are correct it isn't a `unittest` dependency. However if we say move to `pytest`, we would actually have more than one dependency for testing.

Reply:
But they would still be separate dependencies for separate workflows. What's the benefit in combining them into one big list and installing extra stuff?

Reply:
The benefit is that if you are someone who wants to run our tests, that you get all the dependencies required for running said tests. What we do in CI/CD we can document in code; but if a python dev comes along with a fix, they should be able to go `pip install .[test]` and have everything they require to run them.

It's more about our package being correct than being helpful for ci. Though we could add a separate stanza for ci only packages if we want to be clear about it.

Reply:
Hmm, mypy isn't a test, though, in the sense you're talking about.

Reply:
The new `tox.ini` file specifies the test dependencies in the `deps =` line.